pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "niimprint"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
I don't think this is the right place for bumping a version - that should probably be a separate commit with a GitHub release attached to it